vulture_os/portal/views/logon.py

```diff
@@ -566,31 +568,43 @@ def openid_userinfo(request, portal_id=None, workflow_id=None):
         ret = {}
 
         ## JWT ##
-        try:
-            jwt_unverified = jwt.decode(jwt=token, options={"verify_signature": False, "verify_exp": True, "require": ["exp", "iss"]})
-        except (jwt.exceptions.DecodeError, jwt.exceptions.InvalidTokenError) as e:
-            logger.debug(f"PORTAL::openid_userinfo: Token doesn't seem to be a JWT")
-        else:
-            if jwt_unverified:
-                logger.debug("PORTAL::openid_userinfo: Token is a valid JWT")
-                jwt_iss = jwt_unverified["iss"]
-                for connector in OpenIDRepository.objects.filter(enable_jwt=True, provider_url=jwt_iss):
-                    jwt_verified = jwt.decode(jwt=token, algorithms=[connector.jwt_signature_type], key=connector.jwt_key, issuer=connector.provider_url)
-                    if jwt_verified:
-                        logger.debug(f"PORTAL::openid_userinfo: JWT verified")
-                        ret = jwt_verified
-                        break
+        if len(token.split('.')) == 3:
```

Comment:
no, keep try except method to fallback to regular token validation

Reply:
Done